### ISSUE TITLE
firequalizer option 'fft2' and showcqt 'attack' are not found

### ISSUE THREAD

**Issue by flux242** [mfcc64/mpv-scripts#1]
what version of the ffmpeg is compatible with the script? Or maybe some packages are missing on my system? Or maybe the libs should have been compiled with specific options added which wasn't the case with the Ubuntu? Tried to install libavfilter-extra additionally but it didn't help

here is what i have installed:
```
ii  ffmpeg                                 7:3.2.4-1build2                             amd64        Tools for transcoding, streaming and playing of multimedia files
ii  libavcodec57:amd64                     7:3.2.4-1build2                             amd64        FFmpeg library with de/encoders for audio/video codecs - runtime files
ii  libavdevice57:amd64                    7:3.2.4-1build2                             amd64        FFmpeg library for handling input and output devices - runtime files
ii  libavfilter6:amd64                     7:3.2.4-1build2                             amd64        FFmpeg library containing media filters - runtime files
ii  libavformat57:amd64                    7:3.2.4-1build2                             amd64        FFmpeg library with (de)muxers for multimedia containers - runtime files
ii  libavresample3:amd64                   7:3.2.4-1build2                             amd64        FFmpeg compatibility library for resampling - runtime files
ii  libavutil55:amd64                      7:3.2.4-1build2                             amd64        FFmpeg library with functions for simplifying programming - runtime files
ii  libpostproc54:amd64                    7:3.2.4-1build2                             amd64        FFmpeg library for post processing - runtime files
ii  libswresample2:amd64                   7:3.2.4-1build2                             amd64        FFmpeg library for audio resampling, rematrixing etc. - runtime files
ii  libswscale4:amd64                      7:3.2.4-1build2                             amd64        FFmpeg library for image scaling and various conversions - runtime files
```

**Comment by JamesTheAwesomeDude** [comment]
I am getting this as well.

````
[ffmpeg] Parsed_firequalizer_3: Option 'fft2' not found
[ffmpeg] AVFilterGraph: Error initializing filter 'firequalizer' with args 'gain           = 1.4884e8 * f*f*f / (f*f + 424.36) / (f*f + 1.4884e8) / sqrt(f*f + 25122.25):scale          = linlin:wfunc          = tukey:zero_phase     = on:fft2           = on'
parsing the filter graph failed
````
`-v` didn't add any extra informative info; however, here are my ffmpeg details:

````
$ ffmpeg -version 
ffmpeg version 3.2.4 Copyright (c) 2000-2017 the FFmpeg developers
built with gcc 4.9.4 (Gentoo 4.9.4 p1.0, pie-0.6.4)
configuration: --prefix=/usr --libdir=/usr/lib64 --shlibdir=/usr/lib64 --docdir=/usr/share/doc/ffmpeg-3.2.4/html --mandir=/usr/share/man --enable-shared --cc=x86_64-pc-linux-gnu-gcc --cxx=x86_64-pc-linux-gnu-g++ --ar=x86_64-pc-linux-gnu-ar --optflags='-O2 -pipe -march=nehalem -mmmx -msse -msse2 -msse3 -mssse3 -mcrc32 -mcx16 -msahf -mpopcnt -msse4.2 -msse4.1 -mfxsr --param l1-cache-size=32 --param l1-cache-line-size=64 --param l2-cache-size=8192 -mtune=nehalem' --disable-static --enable-avfilter --enable-avresample --disable-stripping --enable-nonfree --disable-indev=oss --disable-indev=jack --disable-outdev=oss --enable-version3 --enable-nonfree --enable-bzlib --disable-runtime-cpudetect --disable-debug --enable-gcrypt --enable-gnutls --enable-gmp --enable-gpl --enable-hardcoded-tables --enable-iconv --enable-lzma --enable-network --enable-openssl --enable-postproc --disable-libsmbclient --enable-ffplay --enable-sdl2 --disable-vaapi --enable-vdpau --enable-xlib --enable-libxcb --enable-libxcb-shm --enable-libxcb-xfixes --enable-zlib --disable-libcdio --disable-libiec61883 --disable-libdc1394 --enable-libcaca --enable-openal --enable-opengl --enable-libv4l2 --enable-libpulse --disable-libopencore-amrwb --disable-libopencore-amrnb --enable-libfdk-aac --enable-libopenjpeg --disable-libbluray --enable-libcelt --disable-libgme --enable-libgsm --disable-mmal --disable-libmodplug --enable-libopus --enable-libilbc --enable-librtmp --disable-libssh --enable-libschroedinger --enable-libspeex --enable-libvorbis --enable-libvpx --disable-libzvbi --enable-libbs2b --disable-chromaprint --enable-libebur128 --disable-libflite --disable-frei0r --disable-libfribidi --enable-fontconfig --enable-ladspa --enable-libass --enable-libfreetype --enable-librubberband --enable-libzimg --enable-libsoxr --enable-pthreads --disable-libvo-amrwbenc --enable-libmp3lame --enable-libkvazaar --disable-nvenc --enable-libopenh264 --disable-libsnappy --enable-libtheora --enable-libtwolame --disable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --disable-amd3dnow --disable-amd3dnowext --disable-aesni --disable-avx --disable-avx2 --disable-fma3 --disable-fma4 --disable-xop --cpu=nehalem --disable-doc --disable-htmlpages --enable-manpages
libavutil      55. 34.101 / 55. 34.101
libavcodec     57. 64.101 / 57. 64.101
libavformat    57. 56.101 / 57. 56.101
libavdevice    57.  1.100 / 57.  1.100
libavfilter     6. 65.100 /  6. 65.100
libavresample   3.  1.  0 /  3.  1.  0
libswscale      4.  2.100 /  4.  2.100
libswresample   2.  3.100 /  2.  3.100
libpostproc    54.  1.100 / 54.  1.100
````

Are we missing packages…is there any extra info we can help provide?

**Comment by mfcc64** [comment]
Generally, you should use latest ffmpeg git master (fft2 option is available since b4e9252ae3d1094b3573299001684ae35d5a5d4f (included in 3.3 release), attack option is available since 3408f4669427f5e950774d135c007f77d0d08d61 (will be included in 3.4 release) . If you cannot do that, the alternative solution is to comment out unavailable options.

Thank's
